docs/docs/snippets/providers/decorators/lazy-inject-declaration.ts

```
@@ -0,0 +1,16 @@
+import {Module} from "@tsed/di";
+
+@Module({
+  // works also with @Injectable
+  imports: [] // Use the imports field if you have services to build
+})
+export default class MyModule {
+  private initialized: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)